ods_ci/tests/Tests/100__deploy/100__installation/110__model_serving_watsonx.robot

```diff
@@ -15,8 +15,9 @@ ${KSERVE_NS}=    ${APPLICATIONS_NAMESPACE}
 *** Test Cases ***
 Verify KServe Is Shipped
     [Documentation]    Verify KServe Is Shipped And Enabled Within ODS
-    [Tags]    WatsonX
+    [Tags]    Smoke
     ...       ODS-2325
+    ...       KServe
```

Comment:
I would prefer that this test was automatically skipped if KServe was not enabled (same as we skip some Monitoring tests for Self-Managed) but if this is the approach you want for now it's ok for me.